.github/workflows/codechecker-analysis.yml

```diff
@@ -28,6 +28,11 @@ jobs:
     steps:
       - name: "Check out repository"
         uses: actions/checkout@v2
+      - name: ccache
+        uses: hendrikmuhs/ccache-action@v1
+        with:
+          key: ccache-ubuntu2004-external-tests
+          max-size: 256M
```

Comment:
Please don't. `CodeChecker log` needs the compiler invocations... I *think* it *can* synergise with CCache, but there are usually some issues cropping up here and there, I'd prefer if it did not intervene.

Reply:
aaah! That is not even meant to be there. haha. Will remove that!